### DISCUSSION TITLE
How to use `initialSelectedKeys` in `useListData` in Select or Combobox

### DISCUSSION THREAD

**Discussion by sadeghbarati** [adobe/react-spectrum#6846] [Q&A] [answered]
I want to select item/items with initialSelectedKeys but it seems not working

I'm seeking for help and thanks for your time 🙏 

https://stackblitz.com/edit/vitejs-vite-cds45n?file=src%2FApp.tsx

**Comment by reidbarber** [comment, marked as answer]
I'm not sure if this is a bug, but I think this is a product of Select/Combobox not supporting multiple selection. You should be able to set the `defaultSelectedKey` on the Select/Combobox themselves to get this to work though.
